src/main/integrations/companion-server/api/v1/index.ts

```diff
@@ -40,6 +40,7 @@ const transformPlayerState = (state: PlayerState) => {
       videoProgress: state.videoProgress,
       volume: state.volume,
       muted: state.muted,
+      shuffled: state.shuffled,
```

Comment:
Considering where the shuffle state actually is I would amend it to the queue object we send right below